README.md

```diff
@@ -190,14 +190,16 @@ scope](https://www.npmjs.com/search?q=scope%3Afirebase) on NPM.
 
 ### Testing the SDK Locally
 
-Please be sure to build your repo before proceeding any further.
+Please be sure to build your SDK changes and install your dependencies before proceeding any further.
```

Comment:
Maybe just say, "run `yarn build` in all packages under test"?

Reply:
`all packages under test` isn't quite correct. I updated it. PTAL